blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_16.py

```diff
@@ -0,0 +1,146 @@
+"""Formula 8.16 from NEN-EN 1992-1-1+C2:2011: Chapter 8: Detailing of reinforcement and prestressing tendons"""
+# pylint: disable=arguments-differ
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011 import NEN_EN_1992_1_1_C2_2011
+from blueprints.codes.formula import Formula
+from blueprints.type_alias import DIMENSIONLESS, MPA
+from blueprints.validations import raise_if_less_or_equal_to_zero, raise_if_negative
+
+
+class Form8Dot16BasicTransmissionLength(Formula):
+    """Class representing formula 8.16 for the calculation of the basic transmission length :math:`l_{pt}`."""
+
+    label = "8.16"
+    source_document = NEN_EN_1992_1_1_C2_2011
+
+    def __init__(
+        self,
+        alpha_1: DIMENSIONLESS,
+        alpha_2: DIMENSIONLESS,
+        diameter: DIMENSIONLESS,
+        sigma_pm0: MPA,
+        f_bpt: MPA,
+    ) -> None:
+        """[:math:`l_{pt}`] Basic value of the transmission length [:math:`mm`].
+
+        NEN-EN 1992-1-1+C2:2011 art.8.10.2.2(2) - Formula (8.16)
+
+        Parameters
+        ----------
+        alpha_1 : DIMENSIONLESS
+            [:math:`α_{1}`] Coefficient taking account of the type of release [-].
+
+            = 1.0 for gradual release;
+
+            = 1.25 for sudden release.
+
+            Use your own implementation for this value or use :class:`SubForm8Dot16Alpha1` class.
+        alpha_2 : DIMENSIONLESS
+            [:math:`α_{2}`] Coefficient taking account of the type of prestressing steel [-].
+
+            = 0.25 for tendons with circular cross-section;
+
+            = 0.19 for 3 and 7-wire strands.
+
+            Use your own implementation for this value or use :class:`SubForm8Dot16Alpha2` class.
+        diameter : DIMENSIONLESS
+            [:math:`Ø`] Nominal diameter of the tendon [:math:`mm`].
```

Comment:
I'm not really familiar with the subject. This seems contradictory to me.
From what I can understand from the context, I think diameter is not DIMENSIONLESS here. I think the correct type would be MM.

Reply:
That's correct! It is a 'typo' from my side.